tensorpack/cmtf.py

```diff
@@ -250,7 +255,8 @@ def perform_CP(tOrig, r=6, tol=1e-6, maxiter=50, progress=False):
 
     if r > 1:
         tFac = sort_factors(tFac)
-
+    
+    if track_error: track_error(ax,error)
```

Comment:
Don't need to call it at the end since it was called at the end of the fitting loop.